### PR TITLE
Create tables

### PR DESCRIPTION
**Pull request by hiromataba**
In this PR, I did the following :point_down: 
- Create tables in SQL.
- Insert and query data in SQL.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```
Comment (AshakaE) [inline queries.sql]: This SQL query will find `mon` anywhere in between an animal name, we are looking for an animal name that **ends** with `mon`

- [ ] Kindly update the query

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * FROM animals WHERE date_of_birth BETWEEN  '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE escape_attempts < 3;
```
Comment (AshakaE) [inline queries.sql]: Missing condition for neutered animals

- [ ] Kindly update query

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10;
```
Comment (AshakaE) [inline queries.sql]: The value we need to check against is 10.5 and not 10

- [ ] Kindly update query

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * FROM animals WHERE neutered = TRUE;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (AshakaE) [inline queries.sql:10]: This query will include start and end values we are looking for animals between 10.4 and 17.3 (no inclusive)

- [ ] Kindly update query

File: schema.sql [hunk 5]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```
Comment (AshakaE) [inline schema.sql:4]: - [OPTIONAL]
 you can set the ID to auto increment by `id int NOT NULL IDENTITY(1,1)`

File: data.sql [hunk 6]
```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name,date_of_birth,weight_kg,neutered,escape_attempts)
+VALUES ('Agumon','Feb 3, 2020',10.23,TRUE,0);
```
Comment (AshakaE) [inline data.sql:4]: - [ ] Kindly update date to `Feb 2` not `Feb 3`

Thank you 🙂

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @hiromataba  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (hiromataba) [reply queries.sql]: select

I think that I put the condition on line 8 please double check

File: data.sql (hunk 6 above)
Comment (hiromataba) [reply data.sql:4]: date

In the requirements, they wrote Feb 3rd that is why I put Feb 3rd not Feb 2

File: queries.sql (hunk 2 above)
Comment (arslanbisharat) [reply queries.sql]: - [ ] As requested by the previous Code Reviewer add a query that lists the name of all animals that are neutered and have less than 3 escape attempts. Currently, on line 8, the query is checking for only neutered animals. Please make the changes on line 5.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by oxenprogrammer — APPROVED** [review]
Hi @hiromataba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**